frontend/packages/kubevirt-plugin/src/selectors/vm-template/advanced.ts

```diff
@@ -162,13 +157,9 @@ export const getTemplateSizeRequirement = (
     sourceSize = convertToBaseValue(getDataVolumeStorageSize(templateSource.dvTemplate));
   }
 
-  return `${pluralize(
-    getDisks(vm).length + (isCDRom ? 1 : 0) + (isWindowsTemplate(template) ? 1 : 0),
```

Comment:
is it ok to drop the "N disks: ..." here ?

Reply:
Yes, thats on purpose. We want to show only storage requirement, not number of disks anymore